src/styles/apps.css

```diff
@@ -50,6 +50,16 @@
   margin-right: .75rem;
 }
 
+[role=banner] .coz-nav .coz-nav-apps-btns-home {
```

Comment:
VLS (Very Long Selector)

Is `.coz-nav` absolutely required here as `.cozy-nav-apps-btns-home` is already prefixed with `.cozy-nav` ?